code/_helpers/text.dm

```diff
@@ -319,6 +319,18 @@
 			break
 	return copytext_char(t, 1, i) + uppertext(copytext_char(t, i, i + 1)) + copytext_char(t, i + 1)
 
+//Returns a string with the first element of the string uncapitalized, ignoring html-tags
+/proc/uncapitalize(t as text)
+	var/i = 1
+	while(copytext_char(t, i, i + 1) == "<")
+		i = findtext_char(t, ">", i + 1)
+		if(i)
+			i++
+		else
+			i = 2
+			break
+	return copytext_char(t, 1, i) + lowertext(copytext_char(t, i, i + 1)) + copytext_char(t, i + 1)
```

Comment:
This whole algorithm can be reduced down to few calls. Also the name is confusing for what it does, sounds generic. You can also do same with `capitalize`
```suggestion
//Returns a string with the first element of the string uncapitalized, ignoring html-tags
/proc/uncapitalize_first_character(t as text)
	var/index = findtext_char(t, ">")
	if(!index)
	    return t
	return copytext_char(t, 1, index + 1) + lowertext(t[index + 1]) + copytext_char(t, index + 2)
```